plat/drivers/virtio-ring: Downgrade saturated descriptors error

mkroening opened the pull request:
<!--

Thank you for opening a new PR to the Unikraft Open Source Project!  We welcome
new changes, features, fixes, and more!  Please fill in this form to indicate
the status of your PR.  Please ensure you have read the contribution guidelines
before opening a new PR as this will cover the PR process:

  https://unikraft.org/docs/contributing/

-->

### Prerequisite checklist

<!--
Please mark items appropriately:
-->

 - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
 - [ ] Tested your changes against relevant architectures and platforms;
 - [ ] Ran the [`checkpatch.uk`](https://github.com/unikraft/unikraft/blob/staging/support/scripts/checkpatch.uk) on your commit series before opening this PR;
 - [ ] Updated relevant documentation.


### Base target

 - Architecture(s): N/A
 - Platform(s): N/A
 - Application(s): N/A


### Additional configuration

<!--
Please specify any additional configuration which is needed for this feature to
work or any new configuration parameters which are introduced by this PR.  This
will help during the review process.  For example:

 - `CONFIG_LIBUKDEBUG=y`

-->

### Description of changes

<!--
Please provide a detailed description of the changes made in this new PR.
-->


Real-world applications may run into https://github.com/unikraft/unikraft/issues/853 many times per second.
I posted a simple reproduction into the issue.
According to @mschlumpp, this just means that Unikraft is filling up the virtio descriptors faster than the host consumes them.
This commit downgrades the error to a debug message to avoid scaring and to spend less time printing errors instead of doing work.

mkroening committed:
plat/drivers/virtio-ring: Downgrade saturated descriptors error

Real-world applications may run into this error many times per second.
I posted a simple reproduction into the issue.
According to @mschlumpp, this just means that Unikraft is filling up the
virtio descriptors faster than the host consumes them.
This commit downgrades the error to a debug message to avoid scaring and
to spend less time printing errors instead of doing work.

Signed-off-by: Martin Kröning <martin.kroening@eonerc.rwth-aachen.de>
GitHub-Closes: #853

jschlumpp reviewed (approved):
Looks good to me :+1: 

Reviewed-by: Marco Schlumpp <marco@unikraft.io>

mkroening commented:
Fixed a typo in the commit message and the PR title.

StefanJum reviewed (changes requested):
Thank you @mkroening, this looks fine. Please wrap the commit description to ~80 characters per line and use `GitHub-Closes: #853` (with `GitHub` added, without a dot at the end and placed after the sign-off tag).

razvand reviewed (changes requested):
Thanks, @mkroening. Prefer using the `GitHub-Closes: #853` line. Also wrap comment lines to 75 characters, see the Checkpatch comments.

mkroening commented:
Done! I addressed all concerns regarding the commit message.

Mind you, I did not change the formatting of the change itself to preserve consistency, although there is a Checkpatch warning, @razvand. :)

razvand commented:
Thanks, @mkroening , this works. @StefanJum , please approve the PR and add your tag and then I'll finalize it.

StefanJum reviewed (changes requested):
@mkroening please remove the first `Closes unikraft#853.` line.
It's all good besides that.

mkroening commented:
> @mkroening please remove the first `Closes unikraft#853.` line.

Done. I did so in the PR message, but forgot to remove it from the commit message. :)

StefanJum reviewed (approved):
All good now, thank you @mkroening 
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>

razvand reviewed (approved):
Approved-by: Razvan Deaconescu <razvand@unikraft.io>
